imports/ui/components/parties/party_update.html

```diff
@@ -0,0 +1,23 @@
+<template name='partiesUpdate'>
+  {{#Layout template=materializeHeader}}
+    {{collection.title}}
+  {{/Layout}}
+  {{#Layout template=materializeContent}}
```

Comment:
Are we capitalizing `Layout` because the lowercase `layout` was taken? If this is so, please rename it to something ie. `materializeLayout` or whatever. This is for clarity, and also because we want a naming convention for our templates (camel case).